.changeset/config.json

```diff
@@ -10,5 +10,9 @@
   "ignore": [],
   "snapshot": {
     "useCalculatedVersion": true
+  },
+  "privatePackages": {
```

Comment:
All public packages get published automatically. We don't want that for blob-local because it's not published to npm. Instead blob-local is a private package that we can create changesets for. If a changeset is created we will create a release and a tag but not publish